### PR TITLE
Add integration test

### PR DESCRIPTION
**Pull request by vshanthe**
## 📝 Description
As part of Test coverage add test cases for User_info and Database_list 

**What does this PR do and why is this change necessary?**
Test coverage

## ✔️ How to Test
make TEST_ARGS="-v user_info" test
make TEST_ARGS="-v database_list" test
**What are the steps to reproduce the issue or verify the changes?**
make TEST_ARGS="-v user_info" test
Regarding the current bug, please uncomment the code and run the command mentioned above to observe the error. Once the issue is resolved, we can re-comment the code

## 📷 Preview

**If applicable, include a screenshot or code snippet of this change. Otherwise, please remove this section.**

### CODE REVIEW

**Review by lgarber-akamai — APPROVED** [review]
LGTM, tests are passing locally! 
Thanks for adding the `user_info` test case btw, that'll make resolving the bug much easier 🎉

File: tests/integration/targets/user_info/tasks/main.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
+- name: user_info
+  block:
+    - debug:
+       msg: Skipping for now...(https://jira.linode.com/browse/TPT-2939)
```
Comment (ykim-1) [inline tests/integration/targets/user_info/tasks/main.yaml:4]: Is there more graceful way to skip the test?

Comment (vshanthe) [reply tests/integration/targets/user_info/tasks/main.yaml:4]: Nope as of now , same has been followed in other test as well. i.e account_info

**Review by jriddle-linode — APPROVED** [review]
LGTM

**Review by ykim-1 — APPROVED** [review]
There are some merge conflicts but LGTM. GJ!

**Pull request merged**